plugins/biometric/build.rs

```diff
@@ -5,9 +5,16 @@
 const COMMANDS: &[&str] = &["authenticate", "status"];
 
 fn main() {
-    tauri_plugin::Builder::new(COMMANDS)
+    if let Err(error) = tauri_plugin::Builder::new(COMMANDS)
         .global_api_script_path("./api-iife.js")
         .android_path("android")
         .ios_path("ios")
-        .build();
+        .try_build()
+    {
+        println!("{error:#}");
```

Comment:
leftover?

Reply:
also when not just ignore error when building for docsrs and otherwise just unwrap and panic.

Reply:
pushed a change to match this @amrbashir